packages/airnode-ois/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { oisSchema } from '@api3/airnode-validator';
 import { OIS } from './types';
 
-export * from '@api3/airnode-validator/dist/src/ois';
+export * from '@api3/airnode-validator/dist/cjs/src/ois';
```

Comment:
@amarthadan WDYT? Shall we export all schemas / types from the `index.ts` to avoid this import? (I think we should, currently, only the API part is exported).

Reply:
Yes, I think we should.

Reply:
See: https://github.com/api3dao/airnode/pull/1051/commits/4611cd5806be654db574f0d8fef8d025b12da19a

I've used named exports for the schemas, since the important validator functions are all implemented in the `api` file. This also means, that the consumers who import the schemas from validator have to use `import {ois} from '@api3/airnode-validator'` and use `ois.someScheme` but I don't think this is that big of a deal.

Reply:
Discussed and agreed upon on slack.